test/CMakeLists.txt

```diff
@@ -157,12 +212,12 @@ set(test_configs "bufr2netcdf" "script2netcdf" "bufr4backend" "script4backend")
 # ===============================================
 # Set test parameters for satellite observations
 # ===============================================
-set(satobs_types "crisf4" "mtiasi" "atms" "satwnd_amv_abi" "satwnd_amv_seviri" "satwnd_amv_viirs" "satwnd_amv_leogeo")
+set(satobs_types "cris-fsr" "mtiasi" "atms" "satwnd_amv_abi" "satwnd_amv_seviri" "satwnd_amv_viirs" "satwnd_amv_leogeo")
```

Comment:
You know this by now, but for anyone else, the CRIS dump file changes over the years to reflex the product it stores.